docs/sources/enterprise/_index.md

```diff
@@ -58,6 +58,10 @@ Supported auth providers:
 
 [White labeling]({{< relref "white-labeling.md" >}}) allows you to replace the Grafana brand and logo with your own corporate brand and logo. You can also change footer links to point to your custom resources.
 
+## Usage Insights
```

Comment:
Is "Usage Insights" actually used in the UI? If so, does this text match the UI text?

Reply:
No it's the generic feature name, I changed it to sentence case.